packages/sit-onyx/src/components/examples/ComponentShowcase/ComponentShowcase.vue

```diff
@@ -0,0 +1,367 @@
+            <strong>Timothy Nixon</strong>
+          </td>
+          <td>active</td>
+          <td>30/10/26</td>
+          <td class="font--soft">-</td>
+        </tr>
+        <tr>
+          <td>
+            <strong>Will Monroe</strong>
+          </td>
+          <td>finished</td>
+          <td>27/03/26</td>
+          <td class="font--soft">14:56</td>
+        </tr>
+        <tr>
+          <td>
+            <strong>Francis Walsh</strong>
+          </td>
+          <td>ready</td>
+          <td>16/12/26</td>
+          <td class="font--soft">12:05</td>
+        </tr>
+      </OnyxTable>
+
+      <OnyxTextarea
+        v-model="state.textarea"
+        class="grid--textarea"
+        label="Textarea"
+        hide-label
+        placeholder="Enter free text here"
+        :autosize="{ min: 7, max: 14 }"
+      />
+
+      <OnyxToastMessage
+        class="grid--toast"
+        headline="Toast notification"
+        description="Description"
+        :duration="0"
+      />
+
+      <div class="showcase__flex grid--tags">
+        <OnyxBadge>Badge</OnyxBadge>
+
+        <OnyxTag label="Tag" />
+        <OnyxTag label="Tag" color="neutral" />
+        <OnyxTag label="Tag" color="danger" />
+        <OnyxTag label="Tag" color="warning" />
+        <OnyxTag label="Tag" color="success" />
+      </div>
+
+      <OnyxCheckboxGroup
+        v-model="state.checkboxGroup"
+        class="grid--checkboxes"
+        label="Checkbox group"
+        hide-label
+        :options="checkboxGroupOptions"
+        with-check-all
+      />
+
+      <div class="showcase__flex grid--row2">
+        <OnyxAvatarStack>
+          <OnyxAvatar
+            v-for="member in teamMembers"
+            :key="member.name"
+            :label="member.name"
+            :src="member.avatar"
+            size="32px"
+          />
+          <OnyxAvatar label="+3" size="32px" />
+        </OnyxAvatarStack>
+
+        <OnyxRadioGroup
+          v-model="state.radioGroup"
+          label="Radio group"
+          hide-label
+          :options="radioGroupOptions"
+          direction="horizontal"
+        />
+      </div>
+
+      <div class="showcase__flex grid--row3">
+        <OnyxInput label="Input" placeholder="Example input" hide-label />
+        <OnyxIconButton :icon="shareIos" label="Icon button" />
+        <OnyxButton :icon="arrowSmallRight" label="Button" color="neutral" />
+        <OnyxButton :icon="checkSmall" label="Button" />
+        <OnyxStepper v-model="state.stepper" label="Stepper" hide-label placeholder="0" />
+      </div>
+
+      <div class="showcase__flex grid--row4">
+        <!-- empty div is needed for spacing purposes -->
+        <div></div>
+        <OnyxCheckboxGroup
+          label="Checkbox group (skeleton)"
+          hide-label
+          direction="horizontal"
+          :skeleton="4"
+          :options="[]"
+        />
+      </div>
+
+      <OnyxNavBar class="grid--nav" app-name="Nav bar" logo-url="/onyx-logo.svg">
```

Comment:
An idea: make one of the nav buttons with an href, so that a pointer cursor can be shown